frontend/apps/case-entity/src/components/variants/occurrence/cnv-tab.tsx

```diff
@@ -0,0 +1,276 @@
+import { useEffect, useState } from 'react';
+import { PaginationState } from '@tanstack/react-table';
+import { X } from 'lucide-react';
+import useSWR from 'swr';
+
+import { Count, GermlineCNVOccurrence, SavedFilterType, SortBody, SortBodyOrderEnum, Sqon } from '@/api/api';
+import DataTable from '@/components/base/data-table/data-table';
+import VariantIcon from '@/components/base/icons/variant-icon';
+import { Card, CardContent } from '@/components/base/ui/card';
+import { SidebarProvider } from '@/components/base/ui/sidebar';
+import QueryBuilder from '@/components/feature/query-builder/query-builder';
+import UserSavedFiltersProps, { getUserSavedFilters } from '@/components/feature/query-builder/user-saved-filters';
+import { FilterComponent } from '@/components/feature/query-filters/filter-container';
+import { FilterList } from '@/components/feature/query-filters/filter-list';
+import { SidebarGroups } from '@/components/feature/query-filters/sidebar-groups';
+import { AggregateContext } from '@/components/feature/query-filters/use-aggregation-builder';
+import { useI18n } from '@/components/hooks/i18n';
+import { cn } from '@/components/lib/utils';
+import { useConfig } from '@/components/model/applications-config';
+import { QueryBuilderState, resolveSyntheticSqon } from '@/components/model/query-builder-core';
+import { queryBuilderRemote } from '@/components/model/query-builder-core/query-builder-remote';
+import { occurrencesApi } from '@/utils/api';
+
+import { OccurrenceCountInput, useCNVOccurencesCountHelper, useCNVOccurencesListHelper } from '../hook';
+
+import { defaultCNVSettings, getCNVOccurrenceColumns } from './table/cnv-occurrence-table-settings';
+
+const DEFAULT_SORTING = [
```

Comment:
I'm not sure about this default sort. These fields does not exist for cnv

Reply:
I forgot to update them when using the same base for SNV. Do you know what should be our default field to sort CNV ?